nexus-repository-composer-it/src/test/java/org/sonatype/nexus/repository/composer/internal/ComposerProxyIT.java

```diff
@@ -81,16 +110,44 @@ public void unresponsiveRemoteProduces404() throws Exception {
     assertThat(status(proxyClient.get(BAD_PATH)), is(HttpStatus.NOT_FOUND));
   }
 
+  public void retrievePackagesJSONFromProxyWhenRemoteOnline() throws Exception {
```

Comment:
Do we modify these json files as part of pulling them through Nexus Repo? I am guessing yes (saw some absolute URLs in them), so if that's the case I would look at doing some tests that verify they are changed as you'd expect.

Reply:
I hope I did this right in: https://github.com/sonatype-nexus-community/nexus-repository-composer/pull/48/commits/555e54e5d3f7d445a82d3bb32f229dbd22e54d61